tests/test_sensor.py

```diff
@@ -86,7 +86,7 @@ def change_aware(sensor, reading, old_reading):
 
 
 def test_observer_sorting(classic_observer, change_aware_observer):
```

Comment:
```suggestion
def test_observer_sorting(classic_observer, delta_observer):
```
(and you probably need to change the fixture definition too...)

Reply:
Yeah, I forgot to do the change in the test, I did it in the source but not here.